superset-frontend/src/features/alerts/AlertReportModal.tsx

```diff
@@ -776,6 +804,25 @@ const AlertReportModal: FunctionComponent<AlertReportModalProps> = ({
     [],
   );
 
+  const dashboard = currentAlert?.dashboard;
+  useEffect(() => {
+    if (!tabsEnabled) return;
+
+    if (dashboard?.value) {
+      SupersetClient.get({
```

Comment:
What happens if this fails? Should we catch it and display an error message?

Reply:
Added a `.catch()` with `dangerToast`